src/bijectors/ordered.jl

```diff
@@ -9,24 +17,6 @@ A bijector mapping ordered vectors in ℝᵈ to unordered vectors in ℝᵈ.
 """
 struct OrderedBijector <: Bijector end
 
-"""
```

Comment:
Was just looking at the docstring for `OrderedBijector`, and it states that it maps from ordered to unordered, but isn't the opposite the case?

```julia
julia> transform(Bijectors.OrderedBijector(), reverse(1.0:5))
5-element Vector{Float64}:
  5.0
 59.598150033144236
 79.68368695633191
 87.07274305526256
 89.7910248837216

julia> transform(inverse(Bijectors.OrderedBijector()), reverse(1.0:5))
ERROR: DomainError with -1.0:
```

Reply:
Woah, true!